web/lib/api/openapi.ts

```diff
@@ -55,6 +54,7 @@ import {
   WorkersCreateSchema,
 } from "lib/types/worker";
 import { z } from "zod";
+import {Allergy} from "../prisma/client";
```

Comment:
Chce to nastavit něco jako prettier a inspekce na ně, podívám se pak na to někdy